docs/source/contributing/developing-core.md

```diff
@@ -289,21 +289,59 @@ Used by Volto, you can also use it in other JavaScript frameworks and environmen
 ## Supported frameworks
 
 Plone supports several frontend implementations, the main one being Volto as the default frontend and reference React-based implementation.
```

Comment:
```suggestion
Plone supports several frontend implementations.
Volto, a React-based implementation, is the default frontend for Plone 6.
```

Reply:
@stevepiercy I'd like to start dropping the "reference implementation" concept here and there. We talked about it during the sprint, if you think that the wording is not correct and we should find a better one, let's do it, but I'd like to convey this from now on for Volto and Classic.

Reply:
Oh, I thought it was a mistake, not intentional. In that case the term is "reference implementation" and should not have other terms inserted within. I updated my suggestion accordingly.

I added the term to the Documentation glossary in https://github.com/plone/documentation/pull/1656.